visualization/app/codeCharta/ui/codeMap/rendering/postprocessor/customComposer.ts

```diff
@@ -0,0 +1,75 @@
+import { WebGLInfo, WebGLRenderer, WebGLRenderTarget } from 'three'
+import { MaskPass,ClearMaskPass } from "three/examples/jsm/postprocessing/MaskPass"
+import { EffectComposer } from 'three/examples/jsm/postprocessing/EffectComposer'
+import { Pass } from 'three/examples/jsm/postprocessing/Pass'
+
+export class CustomComposer extends EffectComposer {
+    private info: WebGLInfo["render"][] = []
+
+    constructor( renderer: WebGLRenderer, renderTarget?: WebGLRenderTarget ) {
+        super(renderer,renderTarget)
+	}
+
+    getInfo() : WebGLInfo["render"]{
+		const result : WebGLInfo["render"] = { calls : 0,lines : 0, triangles : 0, points : 0, frame: 0}
+
+		for (const info of this.info) {
+			result.calls += info.calls;
+			result.lines += info.lines;
+			result.triangles += info.triangles;
+			result.points += info.points;
+	   	}
+        return result;
+    }
+
+    render( deltaTime?: number ): void {
+        if ( deltaTime === undefined ) {
+			deltaTime = this.clock.getDelta()
+		}
+
+		const currentRenderTarget = this.renderer.getRenderTarget()
+		let maskActive = false
+        let pass : Pass;
```

Comment:
This file has not been processed by prettier. You can run the `npm run format` script in the project root once.